### ISSUE TITLE
Inference example

### ISSUE THREAD

**Issue by Bayesianworld** [aapm-bdsc-ontology-tg/radont#70]
Now that we have a consistent ontology in owl, i.e. Reasoner doesn't throw errors, I want to devise an example that exhibits utility of inference.  I have defined 4 subclasses of "technique"
  > SRS: single fraction (?), in brain
  > SBRT: <= 5 Fx, uses MLC, dose > 6 Gy, inverse planning
  > IMRT: >15 Fx or dose < 6 Gy, uses MLC, inverse planning
  > hypofractionation: 6<Fx<16, can use forward or inverse planning

Will need to add a few things, e.g. brain.  See next issue

**Comment by andredekker** [comment]
One of the things to consider is whether or not this should be in the RTO (as a domain ontology) or in an application ontology. E.g. what is and is not hypo-fractionation may not be consistent across sites/countries and also may change over time.

**Comment by jonathanbona** [comment]
I'd vote to flesh out these use cases in an "application ontology" file
that imports the main RTO OWL file. This approach will also allow us to
avoid pulling anatomy classes into RTO proper.

**Comment by Bayesianworld** [comment]
I agree totally, I was just interested in doing an interesting case for illustration purposes for the paper.  I would like to also illustrate the merging of an application ontology and RTO, but I don't think OORO is  written in OWL.  Is ROO appropriate for such a test?
Mark

________________________________
